client/blocks/reader-related-card-v2/style.scss

```diff
@@ -0,0 +1,431 @@
+// Custom breakpoints needed to match original design
+
+$breakpoint-medium: "( min-width: 661px ) and ( max-width: 730px )";
```

Comment:
i think we want to give these names that include file, since sass doesn't scope them to just this file. maybe something like the mouthful `$reader-related-card-v2-breakpoint-medium`?